manifests/kustomize/base/kustomization.yaml

```diff
@@ -9,103 +8,97 @@ bases:
 - mysql
 - cache
 - cache-deployer
-
 resources:
-  - pipeline-application.yaml
-
+- pipeline-application.yaml
 images:
-  - name: gcr.io/ml-pipeline/api-server
-    newTag: 0.5.1
-  - name: gcr.io/ml-pipeline/persistenceagent
-    newTag: 0.5.1
-  - name: gcr.io/ml-pipeline/scheduledworkflow
-    newTag: 0.5.1
-  - name: gcr.io/ml-pipeline/frontend
-    newTag: 0.5.1
-  - name: gcr.io/ml-pipeline/viewer-crd-controller
-    newTag: 0.5.1
-  - name: gcr.io/ml-pipeline/visualization-server
-    newTag: 0.5.1
-  - name: gcr.io/ml-pipeline/metadata-writer
-    newTag: 0.5.1
-  - name: gcr.io/ml-pipeline/cache-server
-    newTag: 0.5.1
-  - name: gcr.io/ml-pipeline/cache-deployer
-    newTag: 0.5.1
-
+- name: gcr.io/ml-pipeline/api-server
+  newTag: 0.5.1
```

Comment:
remember you update it to 1.0.0-rc1, is it due to code base sync or due to branch not merge back to master?

Reply:
As discussed in chat, @Ark-kun prefers merging back the version only after final 1.0 is cut. I think that makes sense, so rcs won't get version bump to master.